app/helpers/ops_helper/textual_summary.rb

```diff
@@ -125,7 +125,7 @@ def textual_vmdb_tables_most_wasted_space
       :title     => _("Tables with Most Wasted Space"),
```

Comment:
Roman, I see your point. But this constant is really in just one place and we removed more constants like that. Maybe we could replace more of the "5" in that serve basically similar purpose with a single constant in a follow up PR?